lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -510,10 +528,9 @@ def post(self):
         try:
             new_data = self.schema.validate(json_data)
         except SchemaError as e:
-            # The format string here is important as the value we're
-            # trying to convert might contain "{}" brackets that would
-            # be interpreted as formatting commands.
-            self.logger.warning("{}", str(e))
+            self.logger.warning(
+                "{}: {} on {!r}", self.__class__.__name__, str(e), json_data
```

Comment:
How is `self.__class__.__name__` different from and/or better than `klasname`, here?

Reply:
I think I just wrote it out twice before realizing I should capture the name once ... and then I missed fixing this one.

Reply:
Ah; actually, because this is a different method and `POST` has only this one use. I suppose I could pull `klasname` up to an instance variable, but that seemed somehow weird ...